skglm/tests/test_group.py

```diff
@@ -132,6 +132,44 @@ def test_vs_celer_grouplasso(n_groups, n_features, shuffle):
     np.testing.assert_allclose(model.coef_, w, atol=1e-5)
 
 
+@pytest.mark.parametrize("n_groups, n_features, shuffle",
+                         [[15, 50, False]])
+def test_intercept_grouplasso(n_groups, n_features, shuffle):
+    n_samples = 100
+    rnd = np.random.RandomState(42)
+    X, y, _ = make_correlated_data(n_samples, n_features, random_state=rnd)
+
+    grp_indices, grp_ptr, groups = _generate_random_grp(n_groups, n_features, shuffle)
+    weights = abs(rnd.randn(n_groups))
+
+    alpha_max = 0.
```

Comment:
can we define `_alpha_max_group_lasso` in some utils instead of redefining it in the test ? It feels like it would be used in other parts of the codebase